src/tokens/functional/typography/typography.json

```diff
@@ -206,7 +214,8 @@
         "$type": "dimension"
       },
       "lineHeight": {
-        "$value": "calc(16/12)"
+        "$value": 1.3,
```

Comment:
How many integers does Figma support? This would be more like 1.3333 in a browser (I think it goes 4 numbers deep). Should we care? 😅 

I'll comment with the other precise values too and we can decide to take it or ignore it